torch_tvm/fusion_pass.h

```diff
@@ -0,0 +1,7 @@
+#include <torch/csrc/jit/passes/alias_analysis.h>
+#include <torch/csrc/jit/passes/graph_fuser.h>
+#include <torch/csrc/jit/passes/utils/subgraph_utils.h>
+
+void FuseSupportedOps(std::shared_ptr<torch::jit::Graph> graph);
```

Comment:
Actually, is this function TVM specific? If not, can we move it to pytorch repro?

Reply:
that's a good catch.  For the sake of efficiency we can "stage" a lot of simple functionality like this here and upstream it to PyTorch at some point.

I'd like to prioritize flushing out functionality before trying to standardize it into an API.  That being said, I've tried to make it very copy-and-paste-able 😛

Reply:
>very copy-and-paste-able

Lol sounds useful

Reply:
> That being said, I've tried to make it very copy-and-paste-able

That's something that can be landed directly into Pytorch in the spirit of hackability of PyTorch. Lol